docs/source/using/index.adoc

```diff
@@ -6,6 +6,7 @@ This section describes how to manage your Minishift VM and interact with your
 OpenShift cluster. It also helps troubleshooting common issues.
 
 - link:../using/managing-minishift{outfilesuffix}[Managing Minishift]
+- link:../using/proxies{outfilesuffix}[HTTP/HTTPS Proxies]
```

Comment:
> I don't get why it would move under quickstart.

@hferentschik Sorry I do not understand your comment. Not sure if you are commenting my comment. I suggested httpproxy stuff should be moved back to its original doc i.e. to managing minishift doc.